src/errors.rs

```diff
@@ -26,40 +26,23 @@ error_chain! {
                 got,
             )
         }
-        OutputMismatch(cmd: Vec<String>, expected: String, got: String) {
+        OutputMismatch(output_name: String, cmd: Vec<String>, expected: String, got: String) {
```

Comment:
We could also embed the `OutputType` enum directly - but then it must be public and would appear in the docs.